lib/msf/ui/console/command_dispatcher/core.rb

```diff
@@ -1420,8 +1416,7 @@ def cmd_route(*args)
             'Netmask' => { 'MaxWidth' => 17 },
           })
 
-      Rex::Socket::SwitchBoard.each { |route|
-
+      Rex::Socket::SwitchBoard.each.with_index { |route, idx|
```

Comment:
I accidentally left this in. I was using it while playing with the idea of supporting route removal by index. I'll push up a commit removing it in a minute.